Makefile

```
@@ -1,3 +1,10 @@
+init:
+	wehich brew || /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Did you mean `which`?

```suggestion
	which brew || /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

---

I'll save up on a review cycle given the time difference and assume the answer is yes. :) Committed

---

heh thanks :)